task_DEV_1/task_DEV_1/Program.cs

```diff
@@ -7,23 +7,28 @@ class Program
         static void Main(string[] args)
         {
             Console.WriteLine("Enter A number");
-            int inputNumber = int.Parse(Console.ReadLine());
+            int firstNumber = int.Parse(Console.ReadLine());
+            Console.WriteLine("Enter B number");
```

Comment:
Maybe would it be better to write smth like "Enter the first number" and "Enter the second number"?